Whiskey/Functions/Install-WhiskeyTool.ps1

```diff
@@ -74,157 +74,166 @@ function Install-WhiskeyTool
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
-    $mutexName = $InstallRoot
-    if( $DownloadRoot )
-    {
-        $mutexName = $DownloadRoot
-    }
-    # Back slashes in mutex names are reserved.
-    $mutexName = $mutexName -replace '\\','/'
-    $mutexName = $mutexName -replace '/','-'
-    $startedWaitingAt = Get-Date
-    $startedUsingAt = Get-Date
-    Write-WhiskeyDebug -Message ('Creating mutex "{0}".' -f $mutexName)
-    $installLock = New-Object 'Threading.Mutex' $false,$mutexName
-    #$DebugPreference = 'Continue'
-    Write-WhiskeyDebug -Message ('[{0:yyyy-MM-dd HH:mm:ss}]  Process "{1}" is waiting for mutex "{2}".' -f (Get-Date),$PID,$mutexName)
+    Write-WhiskeyDebug '\Install-WhiskeyTool\' -Indent
 
     try
     {
-        try
+        $mutexName = $InstallRoot
+        if( $DownloadRoot )
         {
-            [Void]$installLock.WaitOne()
+            $mutexName = $DownloadRoot
         }
-        catch [Threading.AbandonedMutexException]
-        {
-            Write-WhiskeyDebug -Message ('[{0:yyyy-MM-dd HH:mm:ss}]  Process "{1}" caught "{2}" exception waiting to acquire mutex "{3}": {4}.' -f (Get-Date),$PID,$_.Exception.GetType().FullName,$mutexName,$_)
-            $Global:Error.RemoveAt(0)
-        }
-
-        $waitedFor = (Get-Date) - $startedWaitingAt
-        Write-WhiskeyDebug -Message ('[{0:yyyy-MM-dd HH:mm:ss}]  Process "{1}" obtained mutex "{2}" in {3}.' -f (Get-Date),$PID,$mutexName,$waitedFor)
-        #$DebugPreference = 'SilentlyContinue'
+        # Back slashes in mutex names are reserved.
+        $mutexName = $mutexName -replace '\\','/'

```

Comment:
Colons ok? Perhaps `[^a-z0-9_]` -> `-`

Reply:
Yes. Colons are ok. Do you know how long it took me to figure out slashes weren't allowed? Yikes.